compiler/runtime/Helpers.inc

```diff
@@ -353,6 +353,8 @@ SETVAL(TR_PPCAESDecrypt, TR_FSRH+86)
 SETVAL(TR_PPCVectorLogDouble, TR_FSRH+87)
 SETVAL(TR_PPCquadWordArrayCopy_vsx, TR_FSRH+88)
 SETVAL(TR_PPCforwardQuadWordArrayCopy_vsx, TR_FSRH+89)
+SETVAL(TR_PPCPostP10forwardArrayCopy, TR_FSRH+90)
+SETVAL(TR_PPCPostP10genericArrayCopy, TR_FSRH+91)
```

Comment:
These names should match the names of the helpers:

```suggestion
SETVAL(TR_PPCpostP10ForwardCopy, TR_FSRH+90)
SETVAL(TR_PPCpostP10GenericCopy, TR_FSRH+91)
```

Reply:
names changed.

Reply:
The names still do not match, as the capitalization of the first letter of the names is inconsistent.